tests/Fixer/FunctionNotation/VoidReturnFixerTest.php

```diff
@@ -55,6 +55,20 @@ public function provideFixCases()
                  */
                 function foo($param) {}
             '],
+            ['<?php
+                /**
+                 * @return void|array
```

Comment:
is it legal ?
how could one make function like that?
`null|array` is possible in 7.1, but I don't think that one could mix void with sth